public/json/capslock2command.json

```
@@ -0,0 +1,31 @@
+{
+  "title": "Change caps_lock to command ",
+  "rules": [
+	{
+      "description": "Change caps_lock to Command if pressed with other keys, to escape if pressed alone.",
+      "manipulators": [
+        {
+          "type": "basic",
+          "from": {
+            "key_code": "caps_lock",
+            "modifiers": {
+              "optional": [
+                "any"
+              ]
+            }
+          },
+          "to": [
+            {
+              "key_code": "left_command"
+            }
```

It will be better if you add `lazy` modifier flag [1]. Without it, `command` key will be send even when `caps lock` is pressed alone.

[1] https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/to/lazy/

```suggestion
            {
              "key_code": "left_command",
              "lazy": true
            }
```